documentation/src/docs/pravega-concepts.md

```diff
@@ -11,429 +11,312 @@ You may obtain a copy of the License at
-off so that it accepts no further writes.
-
-Segments covering a contiguous range of the key space can also be merged.  At
-time t3, Segment 2’s range and Segment 5’s range are merged into Segment 6 to
-accommodate a decrease in load on the Stream.
-
-When a Stream is created, it is configured with an Scaling Policy that
-determines how a Stream reacts to changes in its load.  Currently there are
-three kinds of Scaling Policy:
-
-1.  Fixed.  The number of Stream Segments does not vary with load
-
-2.  Size-based.  As the number of bytes of data per second written to the Stream
-    increases past a certain target rate, the number of Stream Segments is
-    increased.  If it falls below a certain level, decrease the number of Stream
-    Segments.
-
-3.  Event-based.  Similar to Size-based Scaling Policy, except it uses the
-    number of Events instead of the number of bytes.
+- A Stream starts at time **_t0_** with a configurable number of Segments.  If the
+rate of data written to the Stream is constant, there will be no change in the number of Segments. 
+
+- At time **_t1_**, the system noted an increase in the ingestion rate and splits **_Segment 1_** into two parts. This process is referred as **Scale-up** Event.
+
+- Before **_t1_**, Events with a Routing Key that hashes to the upper part of the key
+space (i.e., values ranging from **200-399**) would be placed in **_Segment 1_** and those that hash into the
+lower part of the key space (i.e., values ranging from **0-199**) would be placed in **_Segment 0_**.
+
+- After **_t1_**, **_Segment 1_** is split into **_Segment 2_** and **_Segment 3_**. The **_Segment 1_** is sealed and stops accepting writes.  At this point in time, Events with Routing Key **_300_** and _above_ are written to **_Segment 3_** and those between **_200_** and **_299_** would be written into **_Segment 2_**.
+
+- **_Segment 0_** continues accepting the same range of Events as before **_t1_**.  
+
+- Another scale-up Event occurs at time **_t2_**, as **_Segment 0_**’s range of Routing
+Key is split into **_Segment 5_** and **_Segment 4_**. Also at this time, **_Segment 0_** is sealed
+and allows no further writes.
+
+- Segments covering a contiguous range of the key space can also be merged. At
+time **_t3_**, **_Segment 2_**’s range and **_Segment 5_**’s range are merged to **_Segment 6_** to
+accommodate a decrease in the load on the Stream.
+
+When a Stream is created, it is configured with a **Scaling Policy** that
+determines, how a Stream handles the varying changes in its load? In the present scenario, Pravega has three kinds of Scaling Policy:
+
+1.  **Fixed**:  The number of Stream Segments does not vary with load.
+
+2.  **Size-based**:  A target rate is set, to decide on increasing or decreasing the number of Stream Segments. If the number of bytes of data per second written to the Stream increases beyond the threshold or target rate, the number of Stream Segments is
```

Comment:
I feel that the previous text was not accurate in defining a scaling policy. I propose the following:
`Data-based: Pravega splits a Stream Segment into multiple ones (i.e., Scale-up Event) if the number of bytes per second written to that Stream Segment increases beyond a defined threshold. Similarly, Pravega merges two adjacent Stream Segments (i.e., Scale-down Event) if the number of bytes written to them fall below a defined threshold.`

Reply:
Data-based: Pravega splits a Stream Segment into multiple ones (i.e., Scale-up Event) if the number of bytes per second written to that Stream Segment increases beyond a defined threshold. Similarly, Pravega merges two adjacent Stream Segments (i.e., Scale-down Event) if the number of bytes written to them fall below a defined threshold.